### PR TITLE
Added toast tweaks

### PR DESCRIPTION
**Pull request by Bearbobs**

### CODE REVIEW

File: packages/editor-ui/src/views/NodeView.vue [hunk 1]
```diff
@@ -604,6 +604,11 @@ export default mixins(
 					this.callDebounced('selectAllNodes', 1000);
 				} else if ((e.key === 'c') && (this.isCtrlKeyPressed(e) === true)) {
 					this.callDebounced('copySelectedNodes', 1000);
+					this.$showMessage({
```
Comment (mutdmour) [inline packages/editor-ui/src/views/NodeView.vue:607]: does not seem like the right place to show this toast. There's a debounce function above that debounces multiple calls. Also what if no nodes are selected.

https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/views/NodeView.vue#L852
seems like a better place..

Comment (Bearbobs) [reply packages/editor-ui/src/views/NodeView.vue]: Adding a boolean flag , `isCut` to deduce the operation as the same function is used for cut

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with `n8n@0.153.0`

**Pull request merged**